ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -89,49 +96,118 @@ void RSkComponentTextInput::OnPaint(SkCanvas *canvas) {
 * @return      True if key is handled else false
 */
 
-void RSkComponentTextInput::onHandleKey(rnsKey  eventKeyType, bool* stopPropagation){
-  *stopPropagation=false;
-  RNS_LOG_TODO("update Keymatrics Event Count.");
-  //TODO  update Keymatrics Event Count.
+void RSkComponentTextInput::onHandleKey(rnsKey eventKeyType, bool *stopPropagation) {
+  std::string textString = displayString_;
+  *stopPropagation = false;
   KeyPressMetrics keyPressMetrics;
   TextInputMetrics textInputMetrics;
+  unsigned int textlen = 0;
+  unsigned int curr = 0;
+  eventCount_++;
```

Comment:
event count should be incremented only when we send changeText event right ?

Reply:
on every key press it has to increase the event count in IOS.